docs/ChangeLog.rst

```diff
@@ -16,6 +16,10 @@ Main changes:
   `requests.Session.request <http://docs.python-requests.org/en/master/api/#requests.Session.request>`__.
   [`#166 <https://github.com/MechanicalSoup/MechanicalSoup/pull/166>`__]
 
+* Added method ``StatefulBrowser.download_link``, which will download the
```

Comment:
Should we add `download_link` to the list of methods in the Referer bullet point?